Frameworks/CoreText/CTFont.mm

```diff
@@ -664,12 +664,20 @@ CFArrayRef CTFontCopyFeatureSettings(CTFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
-bool CTFontGetGlyphsForCharacters(CTFontRef font, const UniChar characters[], CGGlyph glyphs[], CFIndex count) {
-    UNIMPLEMENTED();
-    return StubReturn();
+bool CTFontGetGlyphsForCharacters(CTFontRef fontRef, const UniChar characters[], CGGlyph glyphs[], CFIndex count) {
+    if (!fontRef || !characters || !glyphs || count <= 0) {
+        return false;
+    }
+
+    struct __CTFont* font = const_cast<struct __CTFont*>(fontRef);
+    std::vector<uint32_t> chars(characters, characters + count);
+    font->_dwriteFontFace->GetGlyphIndices(chars.data(), count, glyphs);
+
+    // Failed to get all glyphs if any of the indices are 0
+    return std::none_of(glyphs, glyphs + count, [](CGGlyph glyph) { return glyph == 0; });
```

Comment:
That is, change this to return SUCCEEDED(font->_dwriteFontFace->GetGlyphIndices(...)); instead. 

---
In reply to: [79681059](https://github.com/Microsoft/WinObjC/pull/989#discussion_r79681059) [](ancestors = 79681059)